opslevel/resource_opslevel_tag.go

```diff
@@ -149,6 +149,10 @@ func (r *TagResource) Read(ctx context.Context, req resource.ReadRequest, resp *
 	resourceType := opslevel.TaggableResource(planModel.TargetType.ValueString())
 	data, err := r.client.GetTaggableResource(resourceType, resourceId)
 	if err != nil {
+		if (data == nil || data.ResourceId() == "") && opslevel.IsOpsLevelApiError(err) {
```

Comment:
Same as the others - if the "resource" the tag should be on is not found thats an error - were as if the tag itself is not found thats `RemoveResource`